OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -565,10 +565,20 @@ internal sealed class DreamInterfaceManager : IDreamInterfaceManager {
                 return string.Empty;
             }
 
-            if (!element.TryGetProperty(queryValue, out var value))
-                _sawmill.Error($"Could not winget property {queryValue} on {element.Id}");
+            var multiQuery = queryValue.Split(';', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+            if(multiQuery.Length > 1) {
+                var result = "";
+                foreach(var query in multiQuery) {
+                    if (!element.TryGetProperty(query, out var queryresult))
+                        _sawmill.Error($"Could not winget property {query} on {element.Id}");
+                    result += query+"="+queryresult + ";";
```

Comment:
Rename `queryresult` to `queryResult`
```suggestion
                    if (!element.TryGetProperty(query, out var queryResult))
                        _sawmill.Error($"Could not winget property {query} on {element.Id}");
                    result += query+"="+queryResult + ";";
```